proxy.go

```diff
@@ -108,15 +107,27 @@ func unixAddr(uri string) (string, error) {
 	return addr.Host + addr.Path, nil
 }
 
+func logger() *logrus.Entry {
+	return proxyLog.WithFields(logrus.Fields{
+		"name": proxyName,
+		"pid":  os.Getpid(),
+	})
+}
+
 func setupLogger(logLevel string) error {
 	level, err := logrus.ParseLevel(logLevel)
 	if err != nil {
 		return err
 	}
 
-	logrus.SetLevel(level)
+	proxyLog.SetLevel(level)
+
+	hook, err := lSyslog.NewSyslogHook("", "", syslog.LOG_INFO, "")
```

Comment:
The last arg here needs to be `proxyName` - see https://github.com/clearcontainers/proxy/pull/181.